test/lib/y2firewall/widgets/proposal_test.rb

```diff
@@ -24,8 +24,6 @@
 require "y2firewall/widgets/proposal"
 require "y2firewall/proposal_settings"
 
-require "cwm/rspec"
-
 RSpec.shared_examples "CWM::CheckBox" do
```

Comment:
this should go to https://github.com/yast/yast-yast2/blob/master/library/cwm/src/lib/cwm/rspec.rb So please open pull request with this addition there. Thanks

Reply:
https://github.com/yast/yast-yast2/pull/641